pyro/nn/module.py

```diff
@@ -69,6 +69,16 @@ def set(self, name, value):
             self.cache[name] = value
 
 
+def _get_pyro_params(module):
```

Comment:
nit: this would be slightly faster as
```py
if not isinstance(module, PyroModule):
    yield from module._parameters.items()
    return
_pyro_params = module._pyro_params
for ...
    if ...
        if constrained_name in _pyro_params:
            ...
   ...
```
but no need to change :smile: